src/docker/shared.rs

```diff
@@ -323,49 +320,31 @@ pub(crate) fn docker_mount(
         };
 
         if let Ok(val) = value {
-            let mount_path = mount_cb(docker, val.as_ref(), verbose)?;
-            docker.args(&["-e", &format!("{}={}", var, mount_path.to_utf8()?)]);
+            let mount_path = mount_cb(docker, val.as_ref())?;
+            docker.args(&["-e", &format!("{}={}", var, mount_path)]);
             store_cb((val, mount_path));
             mount_volumes = true;
         }
     }
 
     for path in metadata.path_dependencies() {
-        let mount_path = mount_cb(docker, path, verbose)?;
+        let mount_path = mount_cb(docker, path)?;
         store_cb((path.to_utf8()?.to_string(), mount_path));
         mount_volumes = true;
     }
 
     Ok(mount_volumes)
 }
 
-#[cfg(target_os = "windows")]
-fn wslpath(path: &Path, verbose: bool) -> Result<PathBuf> {
-    let wslpath = which::which("wsl.exe")
-        .map_err(|_| eyre::eyre!("could not find wsl.exe"))
-        .warning("usage of `env.volumes` requires WSL on Windows")
-        .suggestion("is WSL installed on the host?")?;
-
-    Command::new(wslpath)
-        .arg("-e")
-        .arg("wslpath")
-        .arg("-a")
-        .arg(path)
-        .run_and_get_stdout(verbose)
-        .wrap_err_with(|| format!("could not get linux compatible path for `{path:?}`"))
-        .map(|s| s.trim().into())
-}
-
-#[allow(unused_variables)]
-pub(crate) fn canonicalize_mount_path(path: &Path, verbose: bool) -> Result<PathBuf> {
+pub(crate) fn canonicalize_mount_path(path: &Path) -> Result<String> {
```

Comment:
it'd be nice to encode this as a string struct, to ensure we handle it properly.

That can be done later.